src/app/mandate/mandate.component.html

```diff
@@ -0,0 +1,145 @@
+<main class="b-container">
+  @if (complete) {
+    <div id="banner-thank-you"></div>
+  }
+
+<!--  @if (!complete && !noAccess && !timedOut) {-->
+<!--    <mat-spinner class="complete-spinner" color="primary"-->
+<!--                 diameter="30" aria-label="Loading donation details"></mat-spinner>-->
+<!--  }-->
+
+<!--  @if (isDataLoaded) {-->
+    <div>
+      @if (false) {
+        <p class="error">
+          Sorry, you don't have access to this page. To keep your information secure we only allow viewing this
+          page temporarily after making a donation. To see donation details please check your email for a donation
```

Comment:
this is talking about a donation, not relevant for a mandate

Reply:
removed!